rust/src/api/endpoint.rs

```diff
@@ -3,21 +3,33 @@ use crate::{apis::endpoint_api, error::Result, models::*, Configuration};
 
 #[derive(Default)]
 pub struct EndpointListOptions {
-    pub iterator: Option<String>,
+    /// Limit the number of returned items
     pub limit: Option<i32>,
-    pub order: Option<Ordering>,
-}
 
-pub struct Endpoint<'a> {
-    cfg: &'a Configuration,
+    /// The iterator returned from a prior invocation
+    pub iterator: Option<String>,
+
+    /// The sorting order of the returned items
+    pub order: Option<Ordering>,
 }
 
 #[derive(Default)]
 pub struct EndpointStatsOptions {
+    /// Filter the range to data starting from this date
+    ///
+    /// RFC3339 date string.
     pub since: Option<String>,
+
+    /// Filter the range to data ending by this date
+    ///
+    /// RFC3339 date string.
     pub until: Option<String>,
```

Comment:
There might be a way to tune it on the clap side, but fyi I think the help text only shows whatever is in the first paragraph (per markdown semantics) so having the blank line here means it'll be omitted.

Reply:
Well, the CLI is gonna have its own types, so we can customize it however we want there :)